build/charts/antrea/values.yaml

```diff
@@ -67,6 +67,26 @@ ovs:
   # configuration).
   hwOffload: false
 
+secondaryNetwork:
+  # -- Enable secondary network configuration in Antrea.
+  enable: false
+  ovs:
```

Comment:
@arunvelayutham @jianjuns one thing that is not clear to me is how should I configure Antrea is I just want secondary network interfaces with SRIOV, and I don't need / want a secondary OVS bridge? Do I just set `enable` to false? In other words, I have trouble figuring out the relationship between these changes (secondary networks with OVS) and the work that has already been done (SRIO interfaces with no OVS).